fairing/training/kubeflow/deployment.py

```diff
@@ -0,0 +1,52 @@
+from kubernetes import client as k8s_client
+
+from ..native import deployment
+
+class KubeflowDeployment(deployment.NativeDeployment):
+
+    def __init__(self, namespace, runs, distribution):
+        super(KubeflowDeployment, self).__init__(namespace, runs)
+        self.distribution = distribution
+
+    def deploy(self):
+        self.backend.create_tf_job(self.namespace, self.job_spec)
+    
+    def generate_job(self, pod_template_spec):
```

Comment:
There should be a way to generate a job config from a configmap, but it can be a different deployment class to be added later.  Ac configmap job spec allows users/admins to customize the spec.

Reply:
Yes I plan on opening a follow-up issue regarding admin config. I am not sure yet what are all the things an admin would want to configure that I think we need to discuss it a bit before implementation.